apps/eslint/index.js

```diff
@@ -16,7 +16,9 @@ module.exports = {
           return (
             item.type === 'ClassDeclaration' ||
             (item.type === 'ExportDefaultDeclaration' &&
-              item.declaration.type === 'ClassDeclaration')
+              item.declaration.type === 'ClassDeclaration') ||
+            (item.type === 'ExportNamedDeclaration' &&
+              item.declaration?.type === 'ClassDeclaration')
```

Comment:
For my education, why do we need to use `?.`?  When is `item.declaration` null or undefined?

Reply:
You know, I added this based on an error that I ran into and didn't dig into which case was causing this. I figured maybe we had named string exports or something